tools/cassandra_delete_range/internal/cass/cass.go

```diff
@@ -376,7 +343,67 @@ func (c *ClioCass) pruneData(
 		marker.ExitTable()
 	}
 
-	// TODO: take care of nft tables and other stuff like that
+	// nf_token queries
+	if !c.settings.SkipNFTokenTable {
+		marker := util.NewMarker(c.settings.Command)
+		if err := marker.EnterTable("nf_tokens"); err != nil {
+			return err
+		}
+
+		log.Println("Generating delete queries for nft tokens table")
+		rowsCount, deleteCount, errCount = c.prepareAndExecuteDeleteQueries(marker, fromLedgerIdx, toLedgerIdx,
+			"SELECT token_id, sequence FROM nf_tokens WHERE token(token_id) >= ? AND token(token_id) <= ?",
+			"DELETE FROM nf_tokens WHERE token_id = ? AND sequence = ?", deleteMethod{deleteGeneral: maybe.Set(true)}, columnSettings{UseBlob: true, UseSeq: true})
```

Comment:
nf_tokens table is index table of nftpage object. I think it should be deleted in the same way of objects table?